src/lesson6/CreditCard.java

```diff
@@ -0,0 +1,36 @@
+package lesson6;
+
+public class CreditCard {
+    private final int accountNumber;
+    private double balance;
+
+
+    public CreditCard(int accountNumber, double balance) {
+        this.accountNumber = accountNumber;
+        this.balance = balance;
+    }
+
+
+    public void plusSum(double sum) {
+        balance += sum;
+
+    }
+
+    public  double minusSum(double sum) {
+        if (sum > balance) {
+            System.out.println("Error. \n Sum more then balance");
+            return 0;
+        } else {
+            balance -= sum;
+            return sum;
```

Comment:
I think, we should return balance

Reply:
?

Reply:
Why this method is stil doesn't return something?

Reply:
the idea itself removed the return. I didn't see it, sorry!